src/components/Form/Location/Location.jsx

```diff
@@ -850,18 +850,18 @@ Location.defaultProps = {
   spinner: false,
   suggestions: false,
   streetLabel: i18n.t('address.us.street.label'),
-  streetPlaceholder: i18n.t('address.us.street.placeholder'),
+  streetPlaceholder: '',
   street2Label: i18n.t('address.us.street2.label'),
   stateLabel: i18n.t('address.us.state.label'),
-  statePlaceholder: i18n.t('address.us.state.placeholder'),
+  statePlaceholder: '',
   cityLabel: i18n.t('address.us.city.label'),
-  cityPlaceholder: i18n.t('address.us.city.placeholder'),
-  zipcodePlaceholder: i18n.t('address.us.zipcode.placeholder'),
+  cityPlaceholder: '',
```

Comment:
I _think_ we can remove all of these default placeholder props entirely, if we don't reference them in the components, we shouldn't need a default.